lib/rules/require-toggle-inside-transition.js

```diff
@@ -20,6 +20,25 @@ function isValidBindAppear(vBindAppear) {
   return true
 }
 
+/**
+ * @param {string[]} directives
+ */
+function createDirectiveList(directives) {
+  let str = ''
+
+  for (const [index, directive] of directives.entries()) {
+    if (index === directives.length - 1) {
+      str += ` or \`v-${directive}\``
+    } else if (index === 0) {
+      str += ` \`v-${directive}\``
+    } else {
+      str += `, \`v-${directive}\``
+    }
+  }
```

Comment:
It feels odd to have the last string part in the first `if` condition. I'd reorder it to have it more readable:

```suggestion
  for (const [index, directive] of directives.entries()) {
    if (index === 0) {
      str += ` \`v-${directive}\``
    } else if (index < directives.length - 1) {
      str += `, \`v-${directive}\``
    } else {
      str += ` or \`v-${directive}\``
    }
  }
```